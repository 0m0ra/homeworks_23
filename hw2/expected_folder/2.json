{
    "min age": 42,
    "max age": 58,
    "avg age": 48,
    "median age": 46,
    "< 2 days": 0,
    "< 1 week": 0,
    "< 1 month": 0,
    "< 6 months": 33.3,
    "> 6 months": 66.7
}