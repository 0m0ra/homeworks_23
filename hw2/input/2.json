{
    "user": {
        "region": "Ekaterinburg",
        "registered": "2014-12-22",
        "last_login": "2011-10-01",
        "email": "user@icloud.com",
        "age": 31
    },
    "user2": {
        "region": "Kazan",
        "registered": "2012-10-30",
        "last_login": "2010-01-21",
        "email": "user2@gmail.com",
        "age": 19
    }
}