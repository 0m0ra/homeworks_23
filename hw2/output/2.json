{"email": {"useryandex.ru": 50.0, "gmail.com": 50.0}, "registered": {"2012-12-24": 50.0, "2022-10-30": 50.0}}