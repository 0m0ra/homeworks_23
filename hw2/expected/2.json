{
   "0-18": 0.0,
   "18-25": 50.0,
   "26-45": 50.0,
   "45-59": 0.0,
   "60+": 0.0,
   "2014": 50.0,
   "2012": 50.0
}